### ISSUE TITLE
Complex PCA - Avoid or Deprecate

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#441]
In #411 I brought in a hack around scikit's `PCA` to admit complex numbers so we could move the class averaging feature along.  (Scikit, along with most Python community tools, does not support complex).

Two ways to address this and get rid of the hack:
- Transform the data going into regular PCA.
- Use reals as suggested in #439 

I am voting for the later, in which case this issue would be to remove `ComplexPCA` and associated code, replacing with `PCA` directly.  This would also open up that PCA component to other standard or experimental implementations.

**Comment by janden** [comment]
If I understood correctly from our last conversation, though, isn't there a non-equivalency issues between real and complex PCA?

**Comment by garrettwrong** [comment]
Yes, I wrote this (issue) up before looking into the Complex PCA.  I was going to update it after we meet, but will add a little now. 

For anyone interested complex PCA techniques seem to be surveyed pretty well by:

Papaioannou 2017
https://spiral.imperial.ac.uk/handle/10044/1/49235

The legacy code skirts around this issue by using the SVD approach, which works fine over complex numbers. (Also in the legacy matlab implementation FSPCA and related code was all complex).

The result of the Bispectrum _is_ complex values, and the imaginary part is important.  I had been tinkering around with the (real) conversions to see if anything useful came out regarding the bispectrum, but haven't uncovered anything yet other than excess arithmetic and different array access patterns.
